runtime/onert/core/src/exec/Execution.cc

```diff
@@ -151,6 +153,24 @@ void Execution::waitFinish()
 
 bool Execution::isFinished(void) const { return finished; }
 
+#ifdef ONERT_TRAIN
+void Execution::train()
+{
+  const auto execs = dynamic_cast<exec::train::TrainableExecutors *>(_executors.get());
```

Comment:
```suggestion
  auto execs = dynamic_cast<exec::train::TrainableExecutors *>(_executors.get());
```

Reply:
(Out of curiosity)
Why should `const` keyword to be removed?

Reply:
Can't `execs` be updated inside `train()`? 
I'm not sure why there is no compilation error. If you changes `const auto` to `const TrainableExecutors *`, you will get a compilation error.